governance/votes/Executive Vote - November 15, 2023.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Template - [Executive Vote] Spark Proxy Spell, Increase Spark Lend Maximum Debt Ceiling, Launch Project Funding, DAO Resolutions, Whistleblower Bounty, Delegate Compensation - November 15, 2023
```

Comment:
I believe it would be more efficient if the title of the compensation included the month, making it easier to search, like "Nov 2023 AD Compensation"